pkg/gui/command_status_panel.go

```diff
@@ -0,0 +1,66 @@
+package gui
+
+import (
+	"fmt"
+
+	"github.com/jesseduffield/gocui"
+	"github.com/scbizu/cmd"
+)
+
+// CommandStatus defines the go-cmd's cmd
+type CommandStatus struct {
+	command *cmd.Cmd
+	gui     *gocui.Gui
+}
+
+// NewCommandStatus new command status
+func NewCommandStatus(c *cmd.Cmd, gui *gocui.Gui) *CommandStatus {
+	return &CommandStatus{
+		command: c,
+		gui:     gui,
+	}
+}
+
+// Update updates the command status panel
+func (cs *CommandStatus) Update(ui *Gui) {
+
+	go func() {
+		for {
+			select {
+			case status := <-cs.command.Stdout:
```

Comment:
what would be an example value for status? Would it be a random string read from the buffer, or a single character etc?

Reply:
also I'm not sure if this is possible but can we call `cs.refreshCommandStatus(cs.gui, "")` after the command has completed so that we have line breaks between the outputs of different commands (utilising the newline added in `refreshCommandStatus`)?

Reply:
1. The value will be the plain git command output, you can check the output within the dev mode by grep `collect\ message`,however, I will rename it with the `cmdOutput` for better understanding . 

2. Do you mean add new line break for different git command? To implement this, I think we'd better to pass a `echo  \n`(shell) to command status panel.

Reply:
yes, that's what I mean. Although we'd still need to know when to pass that command, which would require us to know when the first command has finished, which if we knew, we could just add a newline. I'm happy to leave this issue as too-tricky-to-solve for now, unless you have an idea of how to do it :)